### ISSUE TITLE
Ledger wallet, weird error messages at launch

### ISSUE THREAD

**Issue by Transisto** [spesmilo/electrum#6327]
![image](https://user-images.githubusercontent.com/388716/86497751-dfeb6200-bd50-11ea-8565-e096083a7634.png)

...

![image](https://user-images.githubusercontent.com/388716/86497760-e7127000-bd50-11ea-88bc-0a606dc8611d.png)

...

![image](https://user-images.githubusercontent.com/388716/86497773-f72a4f80-bd50-11ea-94ba-a972b2b397cf.png)

...

![image](https://user-images.githubusercontent.com/388716/86497779-001b2100-bd51-11ea-9ccf-156fc67f2e99.png)

I'll try reproducing the first one, 

the error window title `electrum-4` should be edited.

**Comment by SomberNight** [comment]
Does it work after unplugging the ledger and plugging it back again?

-------- Original Message --------
On 3 Jul 2020, 23:18, Transisto < notifications@github.com> wrote:
￼
...
￼
...
￼
...
￼
I'll try reproducing the first one,
the error window title electrum-4 should be edited.
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.￼
[ { "@context": "http://schema.org", "@type": "EmailMessage", "potentialAction": { "@type": "ViewAction", "target": "[https://github.com](<a href=)/spesmilo/electrum/issues/6327">https://github.com/spesmilo/electrum/issues/6327", "url": "[https://github.com](<a href=)/spesmilo/electrum/issues/6327">https://github.com/spesmilo/electrum/issues/6327", "name": "View Issue" }, "description": "View this Issue on GitHub", "publisher": { "@type": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Comment by Transisto** [comment]
Yes it end up working, I think it could be related to the Ledger going into
standby mode or something.
